op-program/client/l2/engine_backend_test.go

```diff
@@ -257,10 +287,11 @@ func createBlock(t *testing.T, chain *OracleBackedL2Chain) *types.Block {
 		rawTx := &types.DynamicFeeTx{
 			ChainID:   config.ChainID,
 			Nonce:     nonce,
-			To:        &targetAddress,
+			To:        target,
 			GasTipCap: big.NewInt(0),
 			GasFeeCap: parent.BaseFee(),
-			Gas:       21_000,
+			Gas:       210_000,
```

Comment:
Was this intentional?

Reply:
Yeah. The kzg precompile needs more gas than intrinsic.